lib/pymedphys/docs/lib/howto/get-started.rst

```diff
@@ -33,31 +73,31 @@ following in a terminal or command prompt:
 
     pip install pymedphys[user]
 
+This command can be copied into a command prompt on Windows with a "Right Click"
+of the mouse.
```

Comment:
Maybe make clear the right clicking = pasting? E.g.

"You can copy this command directly into a Windows command prompt. You can _Right Click_ to paste."

Reply:
Implemented :slightly_smiling_face: